rhel/SPECS/dbaas-tools.spec

```diff
@@ -34,8 +41,13 @@ BuildRequires:   golang >= 1.13.0
 mkdir -p src/github.com/kubernetes-sigs/
 mv aws-iam-authenticator-%{commit_aws} src/github.com/kubernetes-sigs/aws-iam-authenticator-%{commit_aws}
 
+%setup -T -c -n kubernetes-%{commit_k8s}
+%setup -q -c -a 1 -n kubernetes-%{commit_k8s}
```

Comment:
I wonder why it is `-a 0` above and `-a 1` there

Reply:
```https://rpm-packaging-guide.github.io/#setup

Option -a (which stands for after) expands those sources after entering. Their arguments are source numbers from the spec file preamble.
```
TL;DR: Specifies exact source number, in this case it means to extract Source1(which is K8s)